router/sources/readme.md

```diff
@@ -44,24 +44,19 @@ Names are currently verified following the below rules:
 
 `Domains` and `Subdomains` are represented on-chain in two ways:
 
-1. As an `NFT`, following our Token Standard. When a name is registered, an NFT resource is created and `moved` into the
-   purchaser's account, and possession of the NFT is proof of ownership of the `Domain` or `Subdomain`. This NFT can be
-   bought, sold, or transferred to another account just like any other `Token`. Names are not permanent, however: much
-   like Web2 domains, the registration will expire after a period of time. Names are currently registrable for durations
-   with increments of 1 year, and the price increases both as the number of years registered increases, and number of
-   glyphs decreases. For more information on pricing, please see the [pricing](#pricing) section.
-2. The central `Registry`, that maps a given `DomainNameKey` (which has both an optional subdmain, and a domain) to its
-   registration details, such as the expiry time, the underlying tokens property version, etc.
+As an `NFT`, following our [Digital Asset Standard](https://aptos.dev/standards/digital-asset). When a name is registered, an `NameRecord` object is created with a reference to represent the ownership. This NFT can be
+bought, sold, or transferred to another account just like any other `Token`. Names are not permanent, however: much
+like Web2 domains, the registration will expire after a period of time. Names are currently registrable for durations
+with increments of 1 year, and the price increases both as the number of years registered increases, and number of
+glyphs decreases. For more information on pricing, please see the [pricing](#pricing) section.
 
 ### Setting Domain/Subdomain Addresses
 
-When a `Domain` or `Subdomain` is registered, it is not automatically associated with an account. Instead, the owner of
-the token can set the address that the name points to. This is done by
-calling `domains::set_name_address(subdomain_name: Option<String>, new_address: address)` to set the address of a name.
+When a `Domain` or `Subdomain` is registered through the router, one can set the target address that the name points to, and can gift the name to another account. 
 The owner can set the name to point to any arbitrary address.
 
 The owner can also `clear` the name by
-calling `domains::clear_name_address(subdomain_name: Option<String>, domain_name: String)`. This will remove the address
+calling `router::clear_target_addr(user: &signer, domain_name: String,  subdomain_name: Option<String>)`. This will remove the address
```

Comment:
[](url)
```suggestion
calling `router::clear_target_addr(user: &signer, domain_name: String, subdomain_name: Option<String>)`. This will remove the address
```